src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -343,7 +338,7 @@ private void AutoSuggestBox_QuerySubmitted(AutoSuggestBox sender, AutoSuggestBox
             }
         }
 
-        private const int millisecondsToWait = 200;
+        private const int millisecondsToWait = 20;

```

Comment:
Can we put this in a seperate PR / commit?   This is a small change with big impact that has a different intent to the rest of these changes.

Reply:
Or I'm happy to do that if your in the middle of something.

Reply:
@ryanbodrug-microsoft Sure!!